README.md

```diff
@@ -13,9 +13,8 @@ The compiler repo can be found [here](https://github.com/softdevteam/ykrustc).
 To get started, see
 [this section in the documentation](https://softdevteam.github.io/ykdocs/dev/getting_started.html).
 
-This repo contains two separately-compiled workspaces, and thus the build
-system is a little non-standard. To act recursively on both workspaces, instead
-of running `cargo <target>`, instead execute `cargo xtask <target>`.
+To work with this repository, instead of running `cargo <target>`, instead
```

Comment:
Didn't you say offline that you need to use `cargo build` without `xtask` if you just want to build? Or can I just always use `xtask` in either case?

Reply:
You can do either, but it's simpler to say "always use xtask".

Reply:
Agreed.